SU-SWS/stanford_profile
# D8CORE-2898 Removed the headline field in favor of title field

# READY FOR REVIEW

# Summary
- The headline field did nothing. It was used to populate the title field with duplicate data
- Remove the headline field and replace it with the title field in all displays and views.

# Need Review By (Date)
- 6/3

# Urgency
- low

# Steps to Test
1. create a few news items
2. verify they all look the same as they did.
3. Alternatively just sync the amp testing site `drush sql:sync @amptesting.01live @self; blt drupal:update` and compare node pages

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
